charts/nexus-iq/values.yaml

```diff
@@ -27,7 +27,7 @@ iq:
   secretMountName: #/etc/secret-volume
   env:
     - name: JAVA_OPTS
-      value: "-Djava.util.prefs.userRoot=${SONATYPE_WORK}/javaprefs"
+      value: "-Djava.util.prefs.userRoot=$(SONATYPE_WORK)/javaprefs"
```

Comment:
is this `$( )` notation to be interpreted by the shell or something else? If it's the shell, It looks to me that this change would make it try to _execute_ a binary called SONATYPE_WORK.

Reply:
@jflinchbaugh I tested locally and with this notation, it is setting the proper values for the environment variable in the `deployment.yaml` file. I used this doc as a reference: https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/

Reply:
ah, so it looks like maybe it's applying the env variable at templating time instead of passing the `${}` notation to the shell at runtime?